test/extensions/filters/udp/udp_proxy/udp_proxy_filter_test.cc

```diff
@@ -190,6 +273,33 @@ class UdpProxyFilterTest : public testing::Test {
   std::unique_ptr<TestUdpProxyFilter> filter_;
   std::vector<TestSession> test_sessions_;
   const Network::Address::InstanceConstSharedPtr upstream_address_;
+  const Network::Address::InstanceConstSharedPtr peer_address_;
+  const std::vector<Network::SocketOptionName> transparent_options{ENVOY_SOCKET_IP_TRANSPARENT,
+                                                                   ENVOY_SOCKET_IPV6_TRANSPARENT};
+};
+
+class UdpProxyFilterTestIpv6 : public UdpProxyFilterTest {
+public:
+  UdpProxyFilterTestIpv6()
+      : UdpProxyFilterTestIpv6(
+            Network::Utility::parseInternetAddressAndPort("[2001:db8:85a3::8a2e:370:7334]:443")) {}
```

Comment:
Can you use a const var for these v6 addresses instead of hard coding them in multiple places?

Reply:
Sure. NP :)